pkg/cmd/docgen/diagrams.go

```diff
@@ -1390,7 +1364,7 @@ var specs = []stmtSpec{
 		exclude: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'EXPERIMENTAL_REPLICA'")},
 	},
 	{
-		name:  "show_transaction",
+		name:  "show_transactions_stmt",
 		stmt:  "show_stmt",
```

Comment:
The stmt here should be `show_transactions_stmt`, not `show_stmt`. See https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/parser/sql.y#L5518.

Also, I missed an inline earlier (`"opt_cluster"`).

So the stmtStruct should look like this:
```
{
		name:  "show_transactions_stmt",
		inline: []string{"opt_cluster"},
	},
```

(As you pointed out, the stmt isn't needed if the name and stmt are the same).